addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/CsrfTokenScanRuleUnitTest.java

```diff
@@ -333,6 +340,11 @@ public void shouldHaveValidReferences() {
 
     private HttpMessage createMessage(boolean isInScope)
             throws URIException, HttpMalformedHeaderException {
+        return createMessage(isInScope, false);
+    }
+
+    private HttpMessage createMessage(boolean isInScope, boolean isGet)
```

Comment:
Could also accept the HTTP method instead of a boolean, e.g.
```suggestion
    private HttpMessage createMessage(boolean isInScope, String httpMethod)
```